rust-src/concordium_base/src/smart_contracts.rs

```diff
@@ -143,6 +144,25 @@ impl WasmModule {
         self.serial(&mut hasher);
         ModuleReference::from(<[u8; 32]>::from(hasher.finalize()))
     }
+
+    /// Attempt to read a [`WasmModule`] from a file.
+    pub fn from_file(path: &std::path::Path) -> anyhow::Result<Self> {
```

Comment:
Since this will be in the Rust SDK, could we avoid exposing `anyhow`? Don't remember how much we are doing this already.

Reply:
I was thinking about that, but then decided not to do that since ParseResult = anyhow::Result already and we have IOErrors.

But I guess in some cases it might be useful to distinguish "read file error" vs. "parse error" even if the latter is just anyhow::Error for now.